react/src/components/ImageEnvironmentSelectFormItems.tsx

```diff
@@ -213,6 +216,15 @@ const ImageEnvironmentSelectFormItems: React.FC<
     } else if (baiClient._config.allow_manual_image_name_for_session) {
       // if no image is available, only set manual if it's allowed
       form.setFieldValue(['environments', 'manual'], version);
```

Comment:
Also I found that there's an another case when the defaultSessionEnvironment is set, but it's not installed or available for now, and the configuration doesn't allow manual image. We didn't thought the case, but I think it's possible. Therefore, when it comes to having a situation like this, we need to apply the first environment and version from the list.

Reply:
But this case already handled in https://github.com/lablup/backend.ai-webui/blob/ef0adef4e521a96dbb2b0dd6a4c3eb576a03e3a8/react/src/components/ImageEnvironmentSelectFormItems.tsx#L195-L209.
I pushed a commit to remove a duplicated case.